tpp-app/src/services/utils/helpers.js

```diff
@@ -44,7 +43,6 @@ export const getDateString = (date) => {
  * @return {boolean} if date is valid and not in the future
  */
  export const isValidDate = (day, month, year) => {
```

Comment:
I made some changes to `isValidDate` on my branch btw, so let me know if you were using it.